### PR TITLE
switch to doing injection of opencontrol data

### LINKED ISSUE #18: rewrite to inject content rather than tags
See https://github.com/opencontrol/compliance-masonry/issues/140#issuecomment-227958235 for background. Basically, template tags aren't flexible enough to handle all the places we need to inject information into the document (e.g. [checkboxes](https://github.com/opencontrol/fedramp-templater/issues/10)). We should flip the data flow so that this codebase handles inserting the data from the OpenControl files directly into the document in a single pass, rather than via an intermediate step of templatizing the Word doc for filling in later.
## Acceptance criteria
- [x] When fedramp-templater is run over an SSP docx, the Responsible Roles (at least) are inserted (rather than the tags)

### PR DESCRIPTION
**Pull request by afeld**
Builds on #23. Closes #18. The more manageable compare view: https://github.com/opencontrol/fedramp-templater/compare/update-deps...inject

### CODE REVIEW

**Comment by afeld** [comment]
After #23 is merged and this is rebased, it will be ready for review! As described in #18, this changes the tool from injecting template tags to injecting actual content from the OpenControl files, using Masonry as a library. (Might mean we want to rename this thing...as a follow-up.) Best part is: it actually works!

SSP screenshot

The hard parts turned out to be
- Wrestling with Go's dependency management
- Getting the testing set up properly

but the code changes are actually relatively straightforward.

**Comment by afeld** [comment]
Rebased and ready for review!

File: opencontrols/data.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	if len(errors) > 0 {
+		return
+	}
+
+	ocd := docx.OpenControlDocx{openControlData}
+	data = Data{ocd}
+	return
+}
+
+// GetResponsibleRoles returns the resposible role information for each component matching the specified control.
```
Comment (jcscottiii) [inline opencontrols/data.go]: `resposible` -> `responsible`

File: opencontrols/data.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package opencontrols
+
+import (
+	"github.com/opencontrol/compliance-masonry/commands/docs/docx"
```
Comment (jcscottiii) [inline opencontrols/data.go:4]: We shouldn't import anything from `commands` folder but we can't do anything right now unless we move all that logic from their to here. that's for another PR though. isn't there a issue for removing docx logic from masonry?

**Comment by jcscottiii** [comment]
@afeld it looks good. just a typo.

There was one thing that i mentioned about what we should import vs not import. But we can't do anything about it yet. See above for the question.

Also, I think we shouldn't have to use the same fixtures from masonry in our unit tests here. But, for now it's all we can do ¯_(ツ)_/¯  lol. I have a created a few issues to refactor masonry later to help with this. Then I created an issue here that references those issues so that once those are closed, we can get away from that. 

The issue I mentioned is here: https://github.com/opencontrol/fedramp-templater/issues/24

**Comment by afeld** [comment]
Yep, totally hear ya...was trying to do this with no changes to Masonry (which I was pleased to see was possible). We can then go back and refactor to make it more library-like.

**Pull request merged**